_includes/fix_analytics_signup_form.html

```diff
@@ -0,0 +1,352 @@
+                        I accept the terms and conditions
+                    {% endif %}
+                </span>
+                <input
+                    required
+                    type="checkbox"
+                    name="accept_terms"
+                />
+            </label>
+            <button type="submit">{% if include.language == "danish" %} Tilmeld {% else %} Signup {% endif %}</button>
+        </div>
+    </form>
+    <div
+        class="contact-form-overlay contact-form-overlay-clickthrough"
+        data-analytics-form-overlay
+    >
+        {% if include.language == "danish" %} Sender... {% else %} Sending... {% endif %}
+    </div>
+</section>
+
+<script>
+    (function () {
+        var form = document.getElementById("contact-form-analytics");
+        var honeypot = form.UZZvvqxJKRlGGbvYaVxz;
+        var nameField = form.name;
+        var emailField = form.email;
+        var currencyField = form.currency;
+        var localeField = form.locale;
+        var customerTypeField = form.customer_type;
+        var locationStreetField = form.location_street;
+        var locationCityField = form.location_city;
+        var locationZipField = form.location_zipcode;
+        var locationStateField = form.location_state;
+        var locationCountryIdField = form.location_country_id;
+        var phoneField = form.phone;
+        var registrationNumberField = form.registration_number;
+        var voucherField = form.voucher;
+        var acceptTermsField = form.accept_terms;
+        var domainField = form.domain_1;
+
+        var contactFormOverlay = document.querySelector("[data-analytics-form-overlay]");
+        var addAdditionalDomainButton = document.querySelector("[data-add-additional-domain");
+        var domainsContainer = document.querySelector("[data-domains-container");
+
+        addAdditionalDomainButton.addEventListener("click", function (e) {
+            e.preventDefault();
+            addNewDomainField();
+        });
+
+        form.addEventListener("submit", function (e) {
+            e.preventDefault();
+            var honeypotValue = honeypot.value;
+            if (honeypotValue !== "") {
+                document.body.append("honeypot: '" + honeypotValue + "' (" + typeof honeypotValue + ")");
+                return;
+            }
+
+            showOverlay();
+
+            var customer = {
+                email: emailField.value,
+                currency: currencyField.value,
+                locale: localeField.value,
+                type: customerTypeField.value,
+                name: nameField.value,
+                countryId: locationCountryIdField.value,
+                street: locationStreetField.value,
+                cityText: locationCityField.value,
+                stateText: locationStateField.value,
+                zipcode: locationZipField.value,
+                phone: phoneField.value,
+                registrationNo: registrationNumberField.value,
+            };
```

Comment:
Basically all of the `xField` values are only used here. No need to extract them, `form.email.value` is just as readable as `emailField.value` imo.

You may be able to make it more readable like this:

```js
var customer = getFormFieldValues(form, ["email", "currency", "locale", "type", "name", ...]);
```

where

```js
function getFormFieldValues(form, fields) {
  var result = {};
  fields.forEach(function (field) {
    result[field] = form[field].value;
  });
  return result;
}
```